app/src/main/java/com/track/finance2up/Finance2UpApplication.kt

```diff
@@ -0,0 +1,16 @@
+package com.track.finance2up
+
+import android.app.Application
+import timber.log.Timber
+
+class Finance2UpApplication: Application() {
+
+    override fun onCreate() {
+        super.onCreate()
+
+        if (BuildConfig.DEBUG){
```

Comment:
code format. Nhớ thêm dấu cách trước dấu `{` nhé. Phần này về sau chắc có ktlint để e đỡ phải làm = tay
```suggestion
        if (BuildConfig.DEBUG) {
```